projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/wpcom-global-styles-view.scss

```diff
@@ -165,71 +170,137 @@
 		transform: translateX(-50%);
 	}
 
-	.launch-bar-global-styles-upgrade,
-	.launch-bar-global-styles-reset {
+	.launch-bar-global-styles-popover__title {
+		font-size: 18px;
+		font-weight: 600;
+		margin: 0;
+	}
+
+	.launch-bar-global-styles__button--upgrade,
+	.launch-bar-global-styles__button--reset {
 		min-height: 28px;
 		box-sizing: border-box;
 		width: 100%;
 		justify-content: center;
 		height: auto;
-	}
-
-	.launch-bar-global-styles-upgrade {
-		border: 1px solid #0675c4;
-		background: #0675c4;
-		color: #fff;
+		display: flex;
+		border: 1px solid #0675c4 !important;
+		background: #0675c4 !important;
+		color: #fff !important;
 		border-radius: 2px;
+		text-decoration: none;
+		align-items: center;
+		padding: 10px;
+		font-size: 13px !important;
+		min-height: 36px !important;
+	}
 
-		&:hover {
-			background: #006ba1;
+	.launch-bar-global-styles__button--reset {
+		background: #fff !important;
+		color: #0675c4 !important;
+		border: none !important;
+		padding: 0;
+		min-height: auto !important;
+		font-weight: 300 !important;
+		svg {
+			fill: #0675c4 !important;
+			margin-right: 6px;
```

Comment:
TODO - look at specificity to clean this up. This issue is that the popover needs to be steeled against incoming global styles from the site. 

So it's either overwrite everything with `*`, shadow DOM or iframe. The latter two being a bit of a faff and probably overkill.